README.md

```diff
@@ -1 +1,2 @@
-# hello-microverse
\ No newline at end of file
+# hello-microverse
+A new project to increase our knowledge about Github , Github Flow and Pull requests. we used linters like lighthouse , webhint , and stylelint. it also contains basic html and css files with basic content like head and body tags as well as styled h1 elements in the header tag. 
```

Comment:
- [ ] **Required:** Please use the template provide by Microverse to create a descriptive README file. Click on the following link for more information [REAMDE](https://github.com/microverseinc/readme-template)